src/session/user.rs

```diff
@@ -146,8 +147,65 @@ mod imp {
 
             let avatar = obj.avatar();
             let user_expression = gtk::ConstantExpression::new(obj);
-            let full_name_expression = super::User::full_name_expression(&user_expression);
-            full_name_expression.bind(avatar, "display-name", gtk::NONE_WIDGET);
+            let display_name_expression = super::User::display_name_expression(&user_expression);
+            display_name_expression.bind(avatar, "display-name", gtk::NONE_WIDGET);
+        }
+    }
+    pub fn time_ago(unix: i32) -> String {
+        let datetime_utc = glib::DateTime::new_now_utc().unwrap().to_unix();
+        let unix: i64 = unix.into();
+        let last_seen = glib::DateTime::from_unix_utc(unix)
```

Comment:
Why are you constructing the DateTime for the second time? Could you not just save the one you constructed in line 155 in a variable?

Reply:
Nevermind, I didn't look close enough.

Reply:
I think now I should use glib::DateTime::difference() instead of my primitive statement. I'll fix it soon